swagger/openapi.json

```
@@ -1271,6 +1271,110 @@
         }
       }
     },
+    "/account/staleness": {
+      "get": {
+        "operationId": "api.account.get_staleness",
```

use `make style` to regenerate this file

```suggestion
        "operationId": "api.account_staleness.get_staleness",
```